engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/InterpreterSpec.scala

```diff
@@ -629,7 +629,7 @@ class InterpreterSpec extends FunSuite with Matchers {
 
     intercept[IllegalArgumentException] {
       interpretSource(process, Transaction())
-    }.getMessage shouldBe "Compilation errors: ErrorValidationParameter(NotBlankParameterValidator,expression,customNode), " +
+    }.getMessage shouldBe "Compilation errors: BlankParameterError(Parameter expression value can't be blank,Please fill expression for this parameter,expression,customNode), " +
```

Comment:
Again, we can use these validators for additional properties or fields, when "expression" does not exist. "value" should be sufficient? (also in EmptyMandatoryParameter, which I think should be renamed to EmptyMandatoryValue)

Reply:
You are right - I've forgotten about it..